.tool-versions

```diff
@@ -1 +1,2 @@
-erlang 24.1.5-3
+erlang 24.1.2
```

Comment:
Q: Why is it downgraded?

Reply:
Oops, accidentally commited this.  I changed it locally just to run things, since this release does not exist for `asdf`.  :see_no_evil: